src/rpc/RPCHelpers.cpp

```diff
@@ -944,11 +948,10 @@ accountHolds(
 )
 {
     ripple::STAmount amount;
-    if (ripple::isXRP(currency)) {
+    if (ripple::isXRP(currency))
         return {xrpLiquid(backend, sequence, account, yield)};
-    }
-    auto key = ripple::keylet::line(account, issuer, currency).key;
 
+    auto key = ripple::keylet::line(account, issuer, currency).key;
```

Comment:
since u have been here, add 'const'

Reply:
added, thanks! There's actually bunch of other functions `transferRate`, `isGlobalFrozen`, and `isFrozen` that has the same issue. But I will leave it for now since this is outside the scope of this PR and to avoid any potential side effects.